src/dmd/backend/ptrntab.d

```diff
@@ -4799,1008 +4799,1014 @@ PTRNTAB2[2] aptb2SHA256MSG2 = /* SHA256MSG2 */ [
 //      The encoding is 0f C0+reg and it is always a 32
 //      bit operation
 
-immutable OP[] optab =
-[
+static immutable OP[] optab = [
```

Comment:
What is `static` doing here?

Reply:
Nothing, it was a leftover from a lambda I used before. Should be good now.